### ISSUE TITLE
[F] Get HTML Table Object by ID and then create workbook Object.

### ISSUE THREAD

**Issue by anurodhsharma11** [exceljs/exceljs#2174]
## 🚀 Feature Proposal

Get already populated <table></table> from webpage by its ID and create workbook object so that further style can be applied and exported to excel.

## Motivation

It is not always the case that we have to recreate the <table></table> already available in webpage from json. Instead we can get reference of the <table></table> from webpage and create workbook object from that. Users can then apply whatever properties and then finally export the xlsx.

## Example

<table id="exampleTable"><thead><tr><th>Sn</th><th>Name</th></tr></thead><tbody><tr><td>1</td><td>Ram Shrestha</td></tr></tbody></table>

const example = document.getElementById("exampleTable");
const wb = new ExcelJS.Workbook();
const sheet = wb.addWorksheet("Sheet1");
sheet.addHtml(example); //---->>such method to transform Html Table to data objects.

**Comment by GergelyDobo** [comment]
Hi @Siemienik,

Is this feature available in version 4.4.0?

**Comment by Siemienik** [comment]
No, it was auto closed as not active

**Comment by GergelyDobo** [comment]
> No, it was auto closed as not active

And are you planning to implement it?

**Comment by Siemienik** [comment]
Indeed, currently, I have other pressing priorities. However, being an open-source initiative, contributions are highly welcome. I'd encourage you to join our Discord community for more detailed discussions. You can also reach out via consulting@siemienik.com to discuss prioritizing features or other collaboration opportunities. Your involvement can indeed accelerate the implementation process.
[![SiemaTeam](https://discordapp.com/api/guilds/976854442009825321/widget.png?style=banner2)](https://discord.gg/siema)
